src/main/scala/org/clulab/habitus/interviews/InterviewsReader.scala

```diff
@@ -0,0 +1,69 @@
+package org.clulab.habitus.interviews
+
+import org.clulab.odin.TextBoundMention
+import org.clulab.utils.Closer.AutoCloser
+import org.clulab.utils.{FileUtils, StringUtils, ThreadUtils}
+import org.clulab.wm.eidos.SimpleEidos
+import org.clulab.wm.eidos.serialization.simple.SimpleSerializer
+
+import java.io.{File, PrintWriter}
+
+object InterviewsReader {
+
+  def main(args: Array[String]): Unit = {
+    val inputDir = "path/to/input/dir"
+    val outputDir = "path/to/output/dir"
+    val threads = 1
+
+    example()
+    run(inputDir, outputDir, threads)
+  }
+
+  def example(): Unit = {
+    val text = "This is a test.  Water trucking in Ethiopia has decreased over August due to the cost of fuel."
+    val eidosSystem = SimpleEidos(useGeoNorm = false, useTimeNorm = false)
+    val annotatedDocument = eidosSystem.extractFromText(text)
+    val simpleSerializer = SimpleSerializer(annotatedDocument)
+
+    simpleSerializer.serialize()
+  }
```

Comment:
This and the call to it can be removed now, probably with some imports as well.

Reply:
@kwalcock, good point, thanks! Done!